### PR TITLE
add what about dash section 

### PR DESCRIPTION
**Pull request by jdamiba**
Relates to https://github.com/plotly/documentation/issues/1654

### CODE REVIEW

File: what-about-dash.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+### What About Dash?
+
+DashR is an open-source framework for building analytical applications, with no Javascript required, and it is tightly integrated with the Plotly graphing library. Everywhere in this page that you see fig, you can display the same figure in a Dash application by passing it to the figure argument of the Graph component from the built-in dashCoreComponents package like this:
```
Comment (nicolaskruchten) [inline what-about-dash.md]: you'll want some links and backticks etc in this paragraph too :)

File: what-about-dash.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+fig <- plot_ly() 
+# fig <- fig %>% add_trace( ... )
+# fig <- fig %>% layout( ... ) 
+
+library(dash)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+
+app <- Dash$new()
+app$layout (
```
Comment (nicolaskruchten) [inline what-about-dash.md]: missing `<-` ?

Comment (nicolaskruchten) [reply what-about-dash.md]: if not, we should remove the space before the `(`

File: what-about-dash.md (hunk 1 above)
Comment (chriddyp) [reply what-about-dash.md]: Also "Dash for R" rather than DashR

**Comment by jdamiba** [comment]
@nicolaskruchten This is ready for review.

File: what-about-dash.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+
+### What About Dash?
+
+[Dash for R](https://dash.plot.ly/) is an open-source framework for building analytical applications, with no Javascript required, and it is tightly integrated with the Plotly graphing library. Everywhere in this page that you see `fig`, you can display the same figure in a Dash for R application by passing it to the `figure` argument of the [`Graph` component](https://dash.plot.ly/dash-core-components/graph) from the built-in `dashCoreComponents` package like this:
```
Comment (chriddyp) [inline what-about-dash.md]:
```suggestion
[Dash for R](https://dashr.plot.ly/) is an open-source framework for building analytical applications, with no Javascript required, and it is tightly integrated with the Plotly graphing library. Everywhere in this page that you see `fig`, you can display the same figure in a Dash for R application by passing it to the `figure` argument of the [`Graph` component](https://dash.plot.ly/dash-core-components/graph) from the built-in `dashCoreComponents` package like this:
```

**Comment by nicolaskruchten** [comment]
As we discovered, we need a link to the installation guide first otherwise people will not know how to install dash :)

File: what-about-dash.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# fig <- fig %>% add_trace( ... )
+# fig <- fig %>% layout( ... ) 
+
+library(dash)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+
+app <- Dash$new()
+app$layout(
+    dccGraph(figure=fig)
```
Comment (nicolaskruchten) [inline what-about-dash.md]:
```suggestion
    htmlDiv(
        list(
                dccGraph(figure=fig) 
        )
     )
```

File: what-about-dash.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+)
+
+app$run_server()
```
Comment (nicolaskruchten) [inline what-about-dash.md]:
```suggestion
app$run_server(debug=TRUE)
```

**Comment by nicolaskruchten** [comment]
Just added the R version of Chris' comments from https://github.com/plotly/documentation/issues/1654

**Comment by jdamiba** [comment]
I have updated the PR with the latest suggestions.

File: what-about-dash.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+library(dash)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+
+app <- Dash$new()
+app$layout(
+    htmlDiv(
+        list(
+                dccGraph(figure=fig)
```
Comment (nicolaskruchten) [inline what-about-dash.md]:
```suggestion
            dccGraph(figure=fig) 
```

**Comment by nicolaskruchten** [comment]
💃 !

**Pull request merged**